setup.py

```diff
@@ -1,18 +1,32 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
 
 setup(
     name="cropharvest",
     description="Open source remote sensing dataset with benchmarks",
-    packages=["cropharvest"],
+    long_description=long_description,
+    author="Gabriel Tseng",
+    author_email="gabrieltseng95@gmail.com",
+    url="https://github.com/nasaharvest/cropharvest",
+    version="0.0.9",
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+    ],
+    packages=["cropharvest"] + [f"cropharvest.{f}" for f in find_packages("cropharvest")],
     install_requires=[
-        "geopandas",
-        "xarray",
-        "earthengine-api",
-        "tqdm",
-        "h5py",
-        "rasterio",
-        "openpyxl",
-        "sklearn",
+        "geopandas==0.9.0",
+        "xarray>=0.16.2",
+        "earthengine-api>=0.1.271",
+        "tqdm>=4.61.1",
+        "h5py>=3.1.0",
+        "rasterio>=1.2.6",
+        "openpyxl>=2.5.9",
+        "scikit-learn>=0.22.2",
```

Comment:
Not specifying versions caused an infinite dependency resolution loop in Google Colab